### ISSUE TITLE
Can't get examples to work.

### ISSUE THREAD

**Issue by Koded** [laconalabs/elliptical#16]
Thanks for this library.

Really want to try it out, but I just can't get any examples working.  

I've tried the first code example in the README.md file:

```

...

// Build our grammar out of Elements
const grammar = (
  <sequence>
    <literal text='flights ' />
    <choice id='direction'>
      <literal text=' from ' value='from' />
      <literal text=' to ' value='to' />
    </choice>
    <Country id='country' />
  </sequence>
)

// Obtain a parse function from our grammar
const parse = compile(grammar)

// Parse based upon a given query
const outputs = parse('flights to irela')
console.log(outputs)
```

but the `outputs` variable is always `[]`.

I've double checked things, and tried other examples, but it is always `[]` that is returned.  

Any advice would be gratefully received.

**Comment by brandonhorst** [comment]
That was my bad. If you look closely, the grammar had two spaces in a row - one after `flights` and one before `from` or `to`. Also, the example used the old `fuzzy={true}` notation for the match strategy, rather than `strategy='fuzzy'`.

Both of these instances were fixed, and I created https://github.com/laconalabs/elliptical-example, which contains an easily runnable demo. Let me know what you think!
